docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -79,8 +85,24 @@ The *Control type* is automatically applied for the field you selected.
 . Click *Save and close*.
 
 [float]
-[[filter-the-data-with-options-lists]]
-==== Filter the data with Options lists
+[[add-time-slider-controls]]
+==== Add time slider controls
+
+To add interactive time slider controls, create the control, then add it to your dashboard.
+
+. Open or create a new dashboard. 
+
+* If you are adding a *Control* to an existing dashboard, click *Edit*, click *Controls* in the dashboard toolbar, then select *Add time slider control*.
+
+* If you are adding a *Control* to a new dashboard, click *Controls* in the dashboard toolbar, then select *Add time slider control*.
+
+. The time slider control uses the time range from the global time filter. To change the time range in the time slider control, <<set-time-filter,change the global time filter>>. 
+
+. Save the dashboard.
```

Comment:
I wonder if you should mention here that saving the dashboard can save the time range selection if a selection is present? It _could_ be a little confusing.

Reply:
Since this section is about adding the control, I left out any information about using the control.